XML3-Homework/PatchNotes.xml

```diff
@@ -0,0 +1,107 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<!-- These are patch notes and future updates of a game I play -->
```

Comment:
@ztd5049 Nice work with this new XML file! I like that you left an XML comment to describe the source of your work. You could actually include that information as metadata in your XML code structure!

Reply:
@ztd5049 Perhaps you might include it in the `<header>` if you include a little more structure, like this:

```xml
<header>
      <title>THIS WEEK AT BUNGIE</title>
      <date isoFormat="2022-01-20">1/20/2022</date>
      <source>Patch Notes and Feature Updates for Bungie Game</source>
      <sourceURL><!--website address here? --></sourceUrl>
      <editor position="communityMgr">A_dmg04</editor>
</header>

Frequently in our XML projects we want to be able to _pull_ information like who edited a file, where it came from, etc , and having a metadata section simply plotted out like that can be super helpful later.